ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__automated-release.yaml

```diff
@@ -74,9 +74,9 @@ tests:
   cron: 8 8 29 2 *
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-autorelease-qe
+    cluster_profile: aws-sustaining-autorelease-412
```

Comment:
4.14 use profile 412.  Is it expected ?

Reply:
Yes, this is expected . We decided to use the same profile for all the EUS release tests jobs.
Since, the actual account name for cloud accounts has the prefix 412, so we didn't want to update the cluster profile name and use it as is.